benchmarking/generate_l1_svm_lp.jl

```diff
@@ -1,11 +1,25 @@
-# Generates a Linear Programming model for an solving L1 SVM problem from the
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generates a Linear Programming model for solving an L1 SVM problem from the
```

Comment:
Since LIBSVM includes non-binary classification problems, but this only builds models for binary ones, change "L1 SVM problem" to "L1 SVM binary classification problem".

Reply:
Done.